x-pack/plugins/cases/public/components/custom_fields/text/get_column.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { EuiTableComputedColumnType } from '@elastic/eui';
+import type { CaseUI } from '../../../containers/types';
+import { getEmptyTagValue } from '../../empty_value';
+
+export const getColumn = ({
```

Comment:
maybe we could rename this to know from just seeing the file name what is it, get_column might be to generic. What about something like get_eui_table_column_props or something similar?